mux_test.go

```diff
@@ -314,7 +315,7 @@ func TestMux(t *testing.T) {
 	// Expect failure for MPP overpayment
 	// TODO(aakselrod): decrypt and check failure message
 	expectResponse(<-responseChan1, 30, routerrpc.ResolveHoldForwardAction_FAIL)
-	require.ErrorIs(t, registry.RequestSettle(testHash), ErrSetNoLongerComplete)
```

Comment:
The commits that rewrite the test can be squashed.

Reply:
I've eliminated this test altogether.